### PR TITLE
Corrected the Contributing Link

### PR DESCRIPTION
**Pull request by souravsingh**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,5 +4,5 @@ PyBee <3's contributions!
 
 Please be aware, PyBee operates under a Code of Conduct. 
 
-See [CONTRIBUTING to PyBee](pybee.org/contributing) for details.
+See [CONTRIBUTING to PyBee](pybee.org/contributing/index.html) for details.
```
Comment (freakboy3742) [inline CONTRIBUTING.md]: Well spotted - but I think the problem is the missing `http://`, not the `index.html` at the end.

Comment (souravsingh) [inline CONTRIBUTING.md]: I checked both ```http://``` and the ```index.html``` links. Both of them work.

Comment (freakboy3742) [inline CONTRIBUTING.md]: If both work, I'd rather use the `http://` - `index.html` is a bit of a messy URL.

BTW: How did you find this repository? I haven't made much of a noise about it yet, and I only pushed it this afternoon!

**Comment by freakboy3742** [comment]
Thanks for the patch! (and thanks for putting up with my fussy preferences! :-)

File: CONTRIBUTING.md (hunk 1 above)
Comment (souravsingh) [inline CONTRIBUTING.md]: I have made the changes accordingly
As for the question, I follow you on Github. So if you do any activity like making a repo, I come to know about it through the feed.

**Pull request merged**